tests/test_utils.py

```diff
@@ -1034,183 +1022,25 @@ def test_operator_application():
-    """Tests for _HFFHeuristic.
-    """
-    initial_state = frozenset({("IsBlock", "block0:block"),
-                               ("IsTarget", "target0:target"),
-                               ("IsTarget", "target1:target"),
-                               ("HandEmpty",),
-                               ("IsBlock", "block1:block")})
-    operators = [
-        utils.RelaxedOperator(
-            "Pick", frozenset({("HandEmpty",), ("IsBlock", "block1:block")}),
-            frozenset({("Holding", "block1:block")})),
-        utils.RelaxedOperator(
-            "Pick", frozenset({("IsBlock", "block0:block"), ("HandEmpty",)}),
-            frozenset({("Holding", "block0:block")})),
-        utils.RelaxedOperator(
-            "Place", frozenset({("Holding", "block0:block"),
-                                ("IsBlock", "block0:block"),
-                                ("IsTarget", "target0:target")}),
-            frozenset({("HandEmpty",),
-                       ("Covers", "block0:block", "target0:target")})),
-        utils.RelaxedOperator(
-            "Place", frozenset({("IsTarget", "target0:target"),
-                                ("Holding", "block1:block"),
-                                ("IsBlock", "block1:block")}),
-            frozenset({("HandEmpty",),
-                       ("Covers", "block1:block", "target0:target")})),
-        utils.RelaxedOperator(
-            "Place", frozenset({("IsTarget", "target1:target"),
-                                ("Holding", "block1:block"),
-                                ("IsBlock", "block1:block")}),
-            frozenset({("Covers", "block1:block", "target1:target"),
-                       ("HandEmpty",)})),
-        utils.RelaxedOperator(
-            "Place", frozenset({("IsTarget", "target1:target"),
-                                ("Holding", "block0:block"),
-                                ("IsBlock", "block0:block")}),
-            frozenset({("Covers", "block0:block", "target1:target"),
-                       ("HandEmpty",)})),
-        utils.RelaxedOperator(
-            "Dummy", frozenset({}), frozenset({}))]
-    goals = frozenset({("Covers", "block0:block", "target0:target"),
-                       ("Covers", "block1:block", "target1:target")})
-    heuristic = _HFFHeuristic(initial_state, goals, operators)
-    assert heuristic(initial_state) == 2
-    assert heuristic(goals) == 0
-    goals = frozenset({("Covers", "block0:block", "target0:target")})
-    heuristic = _HFFHeuristic(initial_state, goals, operators)
-    assert heuristic(initial_state) == 2
-    assert heuristic(goals) == 0
-    # Test unreachable goal.
-    goals = frozenset({("Covers", "block0:block", "target0:target")})
-    heuristic = _HFFHeuristic(initial_state, goals, [])
-    assert heuristic(initial_state) == float("inf")
+        utils.create_heuristic("not a real heuristic", set(), set(), set(),
+                               set(), set())
+    # Cover _Heuristic base class.
```

Comment:
Why did we delete all these tests? Are we not planning to test things like hadd/hmax/hff-lookahead anymore just because we don't have to cover them, since they're no longer in our repo?

Oh, maybe these tests you deleted are on the heuristic themselves which we don't care about, but the lookahead tests are unchanged?

Reply:
right, the lookahead tests are unchanged, this is just testing the heuristics themselves. so really we would just be testing pyperplan's code. I can add back the equivalent tests if you want, but it would be annoying because the inputs are no longer relaxed facts, they're ground atoms.